src/message.lisp

```diff
@@ -49,55 +92,185 @@
        :reader get-message-id)
    (text :initarg :text
          :reader get-text)
+   ;; Caption for file messages
+   (caption :initarg :caption
+            :reader get-caption)
    (chat :initarg :chat
          :reader get-chat)
    (entities :initarg :entities
              :initform nil
              :reader get-entities)
    (raw-data :initarg :raw-data
-             :reader get-raw-data)))
-
-(defclass reply (message)
-  ((reply-to-message :initarg :reply-to-message
-                     :reader get-reply-to-message)))
-
-(defclass forwarded (message)
-  ((forward-from :initarg :forward-from
+             :reader get-raw-data)
+   (forward-from :initarg :forward-from
```

Comment:
What is difference between `forward-from` and `forward-from-chat`? Below I see both of them are filled with `chat` instances.

Probably it is a good moment to add `:documentation` argument to the slots, because now we have so many of them.

Reply:
From Telegram Bot API:
```
forward_from 	        For forwarded messages, sender of the original message
forward_from_chat 	For messages forwarded from channels or from anonymous administrators, information about the original sender chat
```

So the difference is basically the one of whether the forwarded message is initially from a person, or from a chat.

Reply:
Does it mean that the `forward_from` slot should have `private-chat` type?